DevDynamo.Web/Areas/ApiV1/Controllers/ProjectsController.cs

```diff
@@ -52,11 +52,20 @@ public ActionResult<ProjectResponse> Create(CreateProjectRequest request)
             if (!System.IO.File.Exists(path))
             {
                 String[] fileNames = Directory.GetFiles(@"./WorkflowTemplates", "*.txt").Select(fileName => Path.GetFileNameWithoutExtension(fileName)).ToArray();
+                string errorMessage = null! ;
 
-                string allTemplatesNname = (fileNames.Any()) ? string.Join(", ", fileNames.Take(fileNames.Length - 1)) + " and " + fileNames.Last() : fileNames[0];
+                if (!fileNames.Any()) // Folder ./WorkflowTemplates no any file.
+                {
+                    errorMessage = "No any WorkflowTemplate file in this folder.";
+                }
+                else
+                {
+                    string allTemplateNames = (fileNames.Count() > 1) ? string.Join(", ", fileNames.Take(fileNames.Length - 1)) + " and " + fileNames.Last() : fileNames[0];
+                    errorMessage = $"All available templates are {allTemplateNames}.";
```

Comment:
ตรงนี้ผมทดสอบเอาไฟล์ออกให้เหลือแต่ Folder เปล่าๆ แล้วพบว่า error msg แสดงไม่ถูกต้องเป็นแบบนี้ครับ 
` All available templates are  and .` 
ผมเลยขอเพิ่ม `fileNames.Count() > 1` เพื่อให้ข้อความ error msg แสดงผลได้สวยงามครับ 

คิดว่าความเป็นจริงไม่น่าเกิดเคสนี้ขึ้นเพราะ ระบบจะมี Default Template มีไว้เสมอ

Reply:
got it